### PR TITLE
frame_size fixed

### PR DESCRIPTION
**Pull request by uplusplus**
if got_frame is not set, set frame_size to 0.

### CODE REVIEW

**Comment by akallabeth** [comment]
@freerdp-bot test

**Comment by akallabeth** [comment]
@uplusplus what kind of issue does your pull solve?

**Comment by uplusplus** [comment]
In case of len>0 and got_frame==0,
if  frame_size is not assigned to 0,   invalid data will be copied into dst.

File: channels/tsmf/client/ffmpeg/tsmf_ffmpeg.c [hunk 1]
```diff
@@ -428,18 +428,23 @@ static BOOL tsmf_ffmpeg_decode_audio(ITSMFDecoder* decoder, const BYTE *data, UI
...
 			}
+			else
+			{
+				frame_size = 0;
+			}
 			av_free(decoded_frame);
 		}
 #endif
-		if (len <= 0 || frame_size <= 0)
+		if (len> 0)
```
Comment (hardening) [inline channels/tsmf/client/ffmpeg/tsmf_ffmpeg.c]: missing space

**Comment by akallabeth** [comment]
@freerdp-bot test

**Pull request merged**